tests/data/outlet.json

```diff
@@ -0,0 +1 @@
ues": [{"createdTimestampMs": 1606278365225, "id": "388770df-c905-486b-8920-69028a85f42a", "key": "3", "type": "attribute", "updatedTimestampMs": 1606278365225, "value": "1"}], "id": "804376cd-09ca-4c11-9d82-bb6271a6c8fd", "name": "on", "range": {}, "updatedTimestampMs": 1606278197634}, {"createdTimestampMs": 1606276663767, "deviceValues": [{"createdTimestampMs": 1606277377901, "id": "816f3abf-716e-4687-bc7b-07a8d0086d17", "key": "3", "type": "attribute", "updatedTimestampMs": 1606277377901, "value": "0"}], "id": "74c5ad3e-2e96-4fc4-8dfb-09d726d7eebd", "name": "off", "range": {}, "updatedTimestampMs": 1606276663767}]}, {"createdTimestampMs": 1606273033735, "functionClass": "power", "id": "c1ed8865-355e-4f3f-9c15-26d8de2261af", "schedulable": true, "type": "category", "updatedTimestampMs": 1606273033735, "values": [{"createdTimestampMs": 1606274005063, "deviceValues": [{"createdTimestampMs": 1606274217532, "id": "35e9c6db-059d-40cc-9bf5-686862983575", "key": "1", "type": "attribute", "updatedTimestampMs": 1606274217532, "value": "1"}], "id": "d9907860-9b79-464e-823b-9181a523a158", "name": "on", "range": {}, "updatedTimestampMs": 1606274005063}, {"createdTimestampMs": 1606273341335, "deviceValues": [{"createdTimestampMs": 1606273382464, "id": "fedabf54-26e2-4a03-acfe-c3f9c90e9c9e", "key": "1", "type": "attribute", "updatedTimestampMs": 1606273382464, "value": "0"}], "id": "0ce1b511-48ca-4cef-b98e-c290044af483", "name": "off", "range": {}, "updatedTimestampMs": 1606273341335}]}, {"createdTimestampMs": 1606279079078, "functionClass": "timer", "functionInstance": "outlet-1", "id": "78c5fcff-43b3-4565-b650-9964ea8c5095", "schedulable": false, "type": "numeric", "updatedTimestampMs": 1606279079078, "values": [{"createdTimestampMs": 1606279636159, "deviceValues": [{"createdTimestampMs": 1606280475984, "id": "f0881977-ee32-4ddb-84bf-22a49787566e", "key": "50", "type": "attribute", "updatedTimestampMs": 1606280475984}], "id": "091c2109-c818-44e6-85f7-f30e38c481df", "name": "timer-1", "range": {"max": 1440, "min": 0, "step": 1}, "updatedTimestampMs": 1606279636159}]}, {"createdTimestampMs": 1606274219109, "functionClass": "toggle", "functionInstance": "outlet-1", "id": "f0965e95-2866-4e0f-8d87-af8701c37fd6", "schedulable": true, "type": "category", "updatedTimestampMs": 1606274219109, "values": [{"createdTimestampMs": 1606274322861, "deviceValues": [{"createdTimestampMs": 1606274386401, "id": "e630ce24-87ee-4d10-aef5-cb4c965c9c90", "key": "2", "type": "attribute", "updatedTimestampMs": 1606274386401, "value": "0"}], "id": "133e4cb8-8d32-48ec-897f-3d216c67a25b", "name": "off", "range": {}, "updatedTimestampMs": 1606274322861}, {"createdTimestampMs": 1606274775571, "deviceValues": [{"createdTimestampMs": 1606275484730, "id": "6efa1c1e-ee95-4b8a-ac9c-e5b7fda2febc", "key": "2", "type": "attribute", "updatedTimestampMs": 1606275484730, "value": "1"}], "id": "e7cdd2e9-5e12-4f57-8bb0-52ba9bad1201", "name": "on", "range": {}, "updatedTimestampMs": 1606274775571}]}]
```

Comment:
At this time, the json file is essentially a copy of `sample_data`. Since I was unsure of how it was used outside the repository, I opted to not move these files into the `tests` directory.